internal/uvm/cpugroups.go

```diff
@@ -2,55 +2,31 @@ package uvm
 
 import (
 	"context"
-	"errors"
-	"fmt"
 
 	"github.com/Microsoft/hcsshim/internal/cpugroup"
 	"github.com/Microsoft/hcsshim/internal/hcs/resourcepaths"
 	hcsschema "github.com/Microsoft/hcsshim/internal/hcs/schema2"
 )
 
 // ReleaseCPUGroup unsets the cpugroup from the VM
+// It does this by setting the VM's cpugroup to the null group ID of 00000000-0000-0000-0000-000000000000
+//
+// Since a VM must be moved to the null group before potentially being added to a different
+// cpugroup, that means there may be a segment of time that the VM's cpu usage runs unrestricted.
 func (uvm *UtilityVM) ReleaseCPUGroup(ctx context.Context) error {
-	groupID := uvm.cpuGroupID
-	if groupID == "" {
+	if uvm.cpuGroupID == "" {
```

Comment:
I know that this was already here, but I'm wondering if we really need this. I think this was initially written with the idea in mind that we'd be providing knobs to manage cpugroups and a VM's enrollment in a given cpugroup. But, since we expect the owner on the host (iow orchestrator) to create and manage cpugroups, I don't think we need to do this or track the cpugroup at all.

Reply:
Sure, if we don't plan on supporting changing this at runtime (for the time being at least) then I can axe this also.

Reply:
Doesn't really make any sense to get rid of the runtime add and leave this in anyways